src/modules/app-db/repositories/public.repository.ts

```diff
@@ -104,7 +105,7 @@ export class PublicRepository {
     });
   }
 
-  async getRandomGalleries(seed: number, from: number = 0, count: number = 1) {
+  async getRandomGalleries(seed: number, from: number = 0, count: number = 1, type: GalleryType = GalleryType.Real) {

```

Comment:
Ideally, the GalleryType.Real would not be passed in as a default value, but rather by a calling function.